influxdb3_telemetry/src/stats.rs

```diff
@@ -103,7 +103,6 @@ impl<T: Default + Num + Copy + NumCast + PartialOrd> Stats<T> {
 /// possibly a higher resolution.
 ///
 /// For eg.
-///
```

Comment:
Removing this line might look odd when rendered, I would suggest using a header instead of "For eg.", i.e.,
```rust
/// # Example
/// Let's say [...]
```